Source/Libraries/openXDA.APIAuthentication/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,6 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("3.0.3.457")]
-[assembly: AssemblyVersion("3.0.3.457")]
-[assembly: AssemblyFileVersion("3.0.3.457")]
+// [assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyFileVersion("4.0.0.0")]
```

Comment:
If the intent is to version this separately from the rest of the projects, you'll need to tweak `openXDA.buildproj` to exclude this file from the automated version increment.

https://github.com/GridProtectionAlliance/openXDA/blob/0c7fbad7f9ecdc0535ced47c86aebc65515bb178/Build/Scripts/openXDA.buildproj#L99-L107